### DISCUSSION TITLE
L'utilité des skeleton screens dans la conception d'interfaces utilisateur

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#232] [Announcements]
# L'utilité des skeleton screens dans la conception d'interfaces utilisateur

l'utilité des skeleton screens pour une meilleur expérience utilisateur

https://zerofiltre.tech/articles/296-l-utilit%C3%A9-des-skeleton-screens-dans-la-conception-d-interfaces-utilisateur

**Comment by imphilippesimo** [comment]
kolopppp
